### PR TITLE
feat: generate tokens and authentication

### PR DESCRIPTION
**Pull request by Galicz555**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,7 @@
 DB_HOST=remotemysql.com
 DB_USER=KI4VpRQy5l
 DB_PASS=kjuQsqSLer
-DB_NAME=KI4VpRQy5l
\ No newline at end of file
+DB_NAME=KI4VpRQy5l
+
+ACCESS_TOKEN_SECRET=0a8d8cdb0e06153c21cc13f7057bc65162ba2c0cf8b0163eb146da8314b469a794b6723a1bd98e96c5a99d3ab89adafb8a23f787e29ba460db8c4661128a1a16
+REFRESH_TOKEN_SECRET=7c01899130d42186164f28d8cd69d5b2c106c2513a0ca8d8409636e280f6f0e7995752295a3937fc49d05621786ae1baea977175e80437a2093f87db609300cc
```
Comment (matyaspeterfi) [inline .env]: .env should be added to .gitignore, I asked

**Review by matyaspeterfi — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (Galicz555) [reply .env]: solved

**Review by matyaspeterfi — APPROVED** [review]

**Review by gbr88 — APPROVED** [review]

File: src/services/authenticationService.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+const jwt = require('jsonwebtoken');
+
+const accTokExp = '5s';
```
Comment (juhaszPeterDev) [inline src/services/authenticationService.js:3]: lets have 10m access token expiry

File: src/services/authenticationService.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+const refTokExp = '5m';
```
Comment (juhaszPeterDev) [inline src/services/authenticationService.js:4]: lets have one day refresh expiry

File: src/services/heroService.js [hunk 4]
```diff
@@ -83,11 +83,8 @@ class HeroService {
 
           equipment.forEach((element) => {
             const sortedModifiers = modifiers.filter(e => e.equipmentId === element.id);
-            sortedModifiers.forEach((e) => {
-              delete e.equipmentId;
-            });
             const activeState = Boolean(Number(element.active));
-            const equipmentItem = new Equipment(element.name, sortedModifiers, element.type, activeState);
+            const equipmentItem = new Equipment(element.id, element.name, sortedModifiers, element.type, activeState);
```
Comment (juhaszPeterDev) [inline src/services/heroService.js:87]: this is out of scope

File: test/get-hero-by-id-test.js [hunk 5]
```diff
@@ -78,12 +91,30 @@ const expectedResponses = [
   },
 ];
 
+describe('POST /login', () => {
```
Comment (juhaszPeterDev) [inline test/get-hero-by-id-test.js:94]: this would be better in a 'before' section of a test

File: test/tokenTest.rest [hunk 6]
```diff
@@ -0,0 +1,18 @@
+POST http://localhost:3000/login
```
Comment (juhaszPeterDev) [inline test/tokenTest.rest:1]: whats this file for?

**Review by juhaszPeterDev — APPROVED** [review]
